pkgs/applications/graphics/mandelbulber2/default.nix

```diff
@@ -0,0 +1,54 @@
+{ lib, mkDerivation, fetchFromGitHub, libpng, gsl, libsndfile, lzo,
+qmake, qttools, qtbase, qtmultimedia,
+opencl ? true, opencl-clhpp ? null, ocl-icd ? null }:
+
+assert opencl -> opencl-clhpp != null;
+assert opencl -> ocl-icd != null;
+
+mkDerivation rec {
+  pname = "mandelbulber2";
+  version = "2.19";
+
+  src = fetchFromGitHub {
+    owner = "buddhi1980";
+    repo = "mandelbulber2";
+    rev = "refs/tags/${version}";
+    sha256 = "10kq30mxryq78kgsrqwpmws4knz7yfzhdq4zhl28rrz2k1bydgij";
+  };
+
+  nativeBuildInputs = [ qmake qttools ];
+  buildInputs = [ qtbase qtmultimedia libpng gsl libsndfile lzo ] ++
+    lib.optionals opencl [ opencl-clhpp ocl-icd ];
+
+  sourceRoot = "source/mandelbulber2";
+
+  qmakeFlags = [ (if opencl
+                  then "qmake/mandelbulber-opencl.pro"
+                  else "qmake/mandelbulber.pro") ];
+
+  preBuild = ''
+    substituteInPlace "src/system.cpp" --replace "/usr" "$out"
```

Comment:
Can you please open an upstream issue about the hardcoded prefix and add a comment here with a link.